media/themes/Lautus/lautus.msstyles/CMakeLists.txt

```
@@ -1,3 +1,6 @@
+if(DLL_EXPORT_VERSION GREATER_EQUAL 0x600)
```

Please keep an empty line at the CMake file start (dunno why we use these, but better not break it).
```suggestion

if(DLL_EXPORT_VERSION GREATER_EQUAL 0x600)
```